draft-ietf-quic-datagram.md

```diff
@@ -249,11 +249,17 @@ acknowledgements is not used for loss recovery.
 
 If a sender detects that a packet containing a specific DATAGRAM frame might
 have been lost, the implementation MAY notify the application that it believes
-the datagram was lost. Similarly, if a packet containing a DATAGRAM frame is
-acknowledged, the implementation MAY notify the application that the datagram
-was successfully transmitted and received. Note that, due to reordering, a
-DATAGRAM frame that was thought to be lost could at a later point be received
-and acknowledged.
+the datagram was lost.
+
+Similarly, if a packet containing a DATAGRAM frame is acknowledged, the
+implementation MAY notify the application that the datagram was successfully
```

Comment:
which application? The sender, receiver, or both?

Reply:
I appreciate this text was already like this before the PR. However, IMO it wouldn't hurt to be specific here, the following sentences are. 

This isn't a blocking issue, so go ahead and merge if you disagree.

Reply:
I agree with Lucas, how about:
```suggestion
implementation MAY notify the sender application that the datagram was successfully
```
same for line 251 above.